package.json

```diff
@@ -58,6 +58,7 @@
     "raven": "^0.7.2",
     "raw-loader": "^0.5.1",
     "share": "0.7.27",
+    "share-js": "0.0.3",
```

Comment:
Hey, I saw that this was trying to add share-js. Did someone take out share-js or should that still be there? I'm not sure. @brianjgeiger @sloria